homeassistant/components/kulersky/light.py

```diff
@@ -156,8 +156,8 @@ async def async_update(self):
         self._available = True
         brightness = max(rgbw)
         if not brightness:
-            rgbw_normalized = [0, 0, 0, 0]
+            self._attr_rgbw_color = (0, 0, 0, 0)
         else:
-            rgbw_normalized = [round(x * 255 / brightness) for x in rgbw]
+            rgbw_normalized = tuple(round(x * 255 / brightness) for x in rgbw)
```

Comment:
What is the intermediate variable needed for? Could you just assign to the final variable?

Reply:
But then it would be three lines. I think this is cleaner.